src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -30,7 +30,7 @@ def __init__(
             vae=vae,
             text_encoder=text_encoder,
             tokenizer=tokenizer,
-            unet=unet,
+            unet=unet.to(memory_format=torch.channels_last),
```

Comment:
Think this should not be added here. I could see two options

- Either expose some methods which the use can all to set the `memory_format`
- Or create a optimization document and explain these modifications there and let users adapt it.

wdyt @anton-l @patrickvonplaten